memory-viz/src/memory_model.ts

```diff
@@ -642,26 +682,19 @@ export class MemoryModel {
                 this.item_min_height
             );
 
-            if (!stack_frame) {
```

Comment:
I'm not sure why you deleted this.

Reply:
Hi Professor, I believe these lines (and the ones below) added style attributes to elements that previously had these attributes undefined. I removed them because I thought these attributes should default to those defined in the CSS instead

Reply:
Ah I see, yes that's fine 👍